VTEX - Master Data API - v2.json

```diff
@@ -120,6 +120,264 @@
+                },
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/documentResponse"
+                                },
+                                "example": {
+                                    "Id": "CL-b818cbda-e489-11e6-94f4-0ac138d2d42e",
+                                    "Href": "http://api.vtex.com/my-store-name/dataentities/CL/documents/b818cbda-e489-11e6-94f4-0ac138d2d42e"
+                                }
+                            }
+                        }
+                    }
+                },
+                "deprecated": false
+            },
+            "delete": {
+                "tags": ["Customer profiles"],
+                "summary": "Delete customer profile",
+                "description": "Delete a customer profile.",
+                "operationId": "DeleteCustomerProfile",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    },
+                    {
+                        "$ref": "#/components/parameters/id"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/documentResponse"
+                                },
+                                "example": {
+                                    "Id": "CL-b818cbda-e489-11e6-94f4-0ac138d2d42e",
+                                    "Href": "http://api.vtex.com/my-store-name/dataentities/CL/documents/b818cbda-e489-11e6-94f4-0ac138d2d42e"
+                                }
+                            }
+                        }
+                    }
+                },
+                "deprecated": false
+            }
+        },
+        "/api/dataentities/AD/documents": {
+            "post": {
+                "tags": ["Addresses"],
+                "summary": "Create new customer address",
+                "description": "Create new customer address. \r\n\r\n> You can use this request to create customer addresses according to any `AD` schema. Because of this, you are not restricted to using the fields exemplified below in your requests. But you should be aware of the fields allowed or required for the schemas you are using. Learn more about how to use [Master Data v2 schemas](https:\/\/developers.vtex.com\/vtex-rest-api\/docs\/master-data-schema-lifecycle).",
```

Comment:
```suggestion
                "description": "Creates new customer address. \r\n\r\n> You can use this request to create customer addresses according to any `AD` schema. Because of this, you are not restricted to using the fields exemplified below in your requests. But you should be aware of the fields allowed or required for the schemas you are using. Learn more about how to use [Master Data v2 schemas](https:\/\/developers.vtex.com\/vtex-rest-api\/docs\/master-data-schema-lifecycle).",
```

Reply:
Esse link tá certo mesmo?